tensorrt_llm/quantization/quant_annotation.py

```
@@ -0,0 +1,53 @@
+    return node.meta.get('quant_annotation', None)
+
+def has_quant_annotation(node) -> bool:
+    """Check if a node has quantization annotation"""
+    return get_quant_annotation(node) is not None
+
+def clear_quant_annotation(node):
+    """Remove quantization annotation from a node"""
+    if hasattr(node, 'meta') and 'quant_annotation' in node.meta:
+        del node.meta['quant_annotation']
+
+def copy_quant_annotation(source_node, target_node):
```

Ruff (0.12.2)

First line should end with a period, question mark, or exclamation point

Add closing punctuation

(D415)